bun/whatsapp-with-vonage/src/main.ts

```diff
@@ -0,0 +1,66 @@
+import jwt from "jsonwebtoken";
+import { fetch } from "undici";
+import { getStaticFiles, throwIfMissing } from "./utils.ts";
+
+export default async ({ req, res, log, error }: any) => {
```

Comment:
Can we add basic types here as shown here: https://github.com/appwrite/templates/blob/6dab35f1f221ba075640d93f5a8e374f4fba4349/deno/whatsapp-with-vonage/src/main.ts#L6C1-L12C1